### ISSUE TITLE
Validate drift against a specified terraform state

### ISSUE THREAD

**Issue by punitladtw** [snyk/driftctl#1364]
**Description**
Currently when running `driftctl scan`, it assume the default state or specified states apply to the entire AWS account and region. There may however be cases where the account is used by multiple configurators and they only need to validate that the state(s) specified hasn't drifted

**Example**
Not sure exactly how to do this, but it could look like: 
```sh
$ driftctl scan --from tfstate+sf://my-bucket/terraform/environment.tfstate --only-state
```

**Comment by sjourdan** [comment]
hello @punitladtw, can you confirm your use-case matches the discussion here: https://github.com/snyk/driftctl/discussions/272 

Also, are you interested in knowing: 

1. only if a resource is missing
2. if a resource is missing OR drifted/modified

How often would you use such a scan? (ie. nightly scan from github action, etc.)

I'm asking those questions as we're currently working on supporting this, and we're looking for feedback on use-cases like yours.

**Comment by sjourdan** [comment]
hey again @punitladtw - we just released v0.22.0 with a feature that supports exactly this. 

you can use the new `--only-managed` flag to achieve this behavior. 

⚠️ this is [hungry in API calls](https://docs.driftctl.com/0.22.0/usage/cmd/scan-usage#--deep), a future release will work on optimizing this (read: scope requests). 
⚠️ this is working only for the ['deep mode' supported resources](https://docs.driftctl.com/0.22.0/providers/aws/resources/) 

we'd LOVE your feedback 🙏

**Comment by punitladtw** [comment]
This is super awesome! Thank you. We'll try this on our end and see if it works well for our use case (sounds like it would). 

Just to close the loop on your questions... We are interested in both features actually. For our use case, we're a team that doesn't entirely own the AWS accounts that we use, so being able to scope to just the specific state is really beneficial (and thus not having to ignore a large amount of resources). We do plan to run this scans nightly. 

If there are any issues, I can respond to the original idea. Feel free to close this ticket.
